packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -117,6 +118,7 @@ export const getLocationSearch = (req, opts = {}) => {
  * @return {Promise}
  */
 export const render = async (req, res, next) => {
+    performance.mark(PERFORMANCE_MARKS.totalStart)
```

Comment:
What is the performance implications in the long term of not conditionally making these performance marks based on the query param coming in?

(Think of a future where we have hundreds of performance marks, is it possible that this added up to something more than 20ms?)

Reply:
With hundreds of performance markers, the overhead cost will definitely be more than 20ms. I've seen this with my PLP-performance POC.